### PR TITLE
adding suppression for mean_std and corner case resolution

### PR DESCRIPTION
**Pull request by edwardhuh**

### CODE REVIEW

File: src/zentables/zentables.py [hunk 1]
```diff
@@ -505,10 +505,10 @@ def _internal_freq_table(
         **kwargs,
     ) -> pd.DataFrame:
 
-        if not totals or not subtotals:
-            warnings.warn(
-                "`props` is set. Overriding `totals` and `subtotals` settings."
-            )
+        # if not totals or not subtotals:
```
Comment (khwilson) [inline src/zentables/zentables.py]: Why are you commenting this warning out?

File: src/zentables/zentables.py [hunk 2]
```diff
@@ -867,43 +884,55 @@ def _local_suppression(
     Helper function that applies cell suppression to mini_df_n.
     mini_df_n is assumed to be a smaller portion of a larger
     """
+
+    df = mini_df_n.copy()
+    colnames = mini_df_n.columns
+    rownames = mini_df_n.index
+    df.columns = np.arange(len(colnames))
```
Comment (khwilson) [inline src/zentables/zentables.py:891]: Ugh this is so gross, but it definitely works!

File: src/zentables/zentables.py [hunk 3]
```diff
@@ -867,43 +884,55 @@ def _local_suppression(
...
 
+        if (len(coi) != 0 or len(roi) != 0) and 1 in mask.shape:
+            # Corner Case: if there is one row or column, and one is masked, the rest needs to be masked
+            return pd.DataFrame(
+                np.ones(mask.shape, dtype=bool), columns=colnames, index=rownames
+            )
         if len(coi) == 0 and len(roi) == 0:
             break
 
+        # reassign column names and row names
```
Comment (khwilson) [inline src/zentables/zentables.py]: Wrong indentation

**Review by khwilson — COMMENTED** [review]
Extremely small comment but otherwise 👍

File: src/zentables/zentables.py (hunk 1 above)
Comment (edwardhuh) [reply src/zentables/zentables.py]: It was flooding my terminal, so I temporarily commented it out.
Will uncomment it now!

File: src/zentables/zentables.py (hunk 2 above)
Comment (edwardhuh) [reply src/zentables/zentables.py:891]: Yep 100% agree!

**Pull request merged**